packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -55,6 +55,25 @@ export interface ClusterProps {
   readonly capacityProviders?: string[];
 
   /**
+<<<<<<< Updated upstream
+=======
```

Comment:
let's remove these merge markers 

also on line 76

Reply:
Thank you so much for guiding, I removed those stashed changes, I don't know why these commits appear in my PR when I cherry-pick the latest commit which is related to alarmrollback one